### DISCUSSION TITLE
extending for remote control

### DISCUSSION THREAD

**Discussion by flowernert** [geoffreybennett/alsa-scarlett-gui#93] [Ideas]
Hi Geoffrey

I'm looking forward to extend the great work you made, I'd like to add a remote control feature that could be used from a smartphone or tablet.
I think it would be useful to any sound engineer willing to set the headphones returns mixes of musicians without going back and forth to the record room/control room.

I'd think either 
- adding a listening network socket directly inside your app that you could turn on/off through a button in the main panel
- create an independent server program that would translate remote calls into commands for your app

I'll work on a prototype for this and propose to merge if it is successful.

Do you have any ideas, opinions or specific implementation details you would like to point at beforehand?

**Comment by geoffreybennett** [comment]
Hi @flowernert,

I think remote control is a good idea, but I don't think this software is the right place to do it. Please see issue #92.

Thanks,
Geoffrey.

**Reply by flowernert** [reply]
Thanks, indeed as mentionned in #92 some kind of OSC-Alsa bridge would be the proper place for this

**Comment by johnedstone** [comment]
This github project is awesome. Has there been any development that anyone knows for remote control yet? Thanks again Geoffrey!
